### PR TITLE
Update landing page

### PR DESCRIPTION
**Pull request by natke**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/a4c435a2da56490fb0fdf0b13dcd244119966909/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4bf5d4d-5682-423a-96da-9c7306c7f63c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/70f5b703f4c02d016b02e15ed3d459db06155615/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3cefda4c-b3db-49b2-a121-132409a19867_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/7a975a7198ab34886110e1085d61e2cf4fc2e95f/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1eb025e-2620-4cb7-957d-ad861aa47d53_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1bf4fb83875e4c92c1d6870f0c3d97e77cd886e1/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/55b7bf68-e90d-4d72-a66d-bb4b31e27fba_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1e63a2a9b350e4458a7ca1d6c4144cc058cc5ec2/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9cd3ae58-df2c-44f0-bcdd-b366f91a43fb_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/ee35e0ac94430f956134886c65251306401fe538/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2c2220d-2225-40ca-a766-8aba191051fb_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/index.yml [hunk 1]
```diff
@@ -22,39 +20,63 @@ landingContent:
...
-  # Concept
-  - title: Learn about quantum computing 
+  # Develop
+  - title: Develop with Q# and Python or Jupyter Notebooks
     linkLists:
-      - linkListType: concept
+    - linkListType: how-to-guide
+      links:
+      - text: Run Q# online in Jupyter
+        url: aka.ms/try-quantum-katas
```
Comment (bromeg) [inline articles/index.yml]: this link appears to be broken from the preview. Does it need https:// ?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/8ddc12982bddb1fbbd117472656c799f30149f37/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb511954-cab1-45f4-b51f-cfdffdeba54f_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/56eae0909c21171e40cd972843c30e9b6f91ef0e/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b08d6a0b-0d2e-420e-bd62-cccaef583941_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/TOC.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bea50a364f967a0554c4fa85ca74fbdc8b1e8246/articles/TOC.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5a3ba15-5228-4da0-aa64-6ade8f7a9adb_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/index.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bea50a364f967a0554c4fa85ca74fbdc8b1e8246/articles/index.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/09ccfec4-7223-4895-95a8-8ba052cf1715_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/machines/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bea50a364f967a0554c4fa85ca74fbdc8b1e8246/articles/machines/index.md) | 87 | 71 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ce16dc94-fad8-4d6d-8549-497a396dcc87_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/machines/toc.yml](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bea50a364f967a0554c4fa85ca74fbdc8b1e8246/articles/machines/toc.yml) | 100 | 100 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7e62409e-a661-4979-8eb8-cb8ce51278be_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**